scripts/examples/oracle/oracle.sh

```diff
@@ -15,8 +15,8 @@ elysd query oracle show-price BTC
 elysd query gov proposals
 
 elysd tx oracle request-band-price
-elysd tx oracle add-asset-info-proposal satoshi BTC BTC "" "" --title="title" --description="description" --deposit="10000000000000000000aelys" --from=alice --chain-id=elys --broadcast-mode=block --yes
-elysd tx oracle remove-asset-info-proposal satoshi --title="title" --description="description" --deposit="10000000000000000000aelys" --from=alice --chain-id=elys --broadcast-mode=block --yes
+elysd tx oracle add-asset-info-proposal satoshi BTC BTC "" "" --title="title" --description="description" --deposit="10000000000000000000uelys" --from=alice --chain-id=elys --broadcast-mode=block --yes
+elysd tx oracle remove-asset-info-proposal satoshi --title="title" --description="description" --deposit="10000000000000000000uelys" --from=alice --chain-id=elys --broadcast-mode=block --yes
```

Comment:
@junkai121 you need to reduce the trailing zeros in those amounts as the denom `uelys` uses 6 decimals max